### DISCUSSION TITLE
Link to website if feature is not supported

### DISCUSSION THREAD

**Discussion by JonathanBout** [community/community#86647] [Mobile]
It'd be great if we could click a link to go to the website to use a feature currently not supported by the app, for example a boards project.

Now I have to open my browser, navigate to GitHub.com and search for the project and hope it doesn't switch to the app.

**Comment by elisealix22** [comment]
Hi @JonathanBout , I definitely understand your request and have a couple of follow-up questions! I'm not sure which version of the app you are using, but are you using a more recent one with Projects support? https://github.blog/changelog/2023-03-14-projects-on-github-mobile-is-now-generally-available/

If so, is there a limitation of our Projects support that makes you want to jump out to the website?

**Reply by JonathanBout** [reply]
I just updated and am now on `GitHub Mobile v1.145.0-beta (10171)` but I still get the message that boards layout is not supported.

**Reply by elisealix22** [reply]
Got it! Thank you for clarifying. I opened an internal issue to discuss linking back out to web here for the time being. Thanks for double checking!
